File: web-manager/src/main/webapp/js/controller/seckillOrderController.js
app.controller('seckillOrderController',function ($scope,seckillOrderService,$controller) {
    //继承baseController
    $controller('baseController',{$scope:$scope});
    //查询所有并展示
    $scope.findAll=function() {
        seckillOrderService.findAll().success(
            function (response) {
                $scope.list = response;
            }
        )
    }

    //分页查询
    $scope.findPage=function(page,rows){
        seckillOrderService.findPage(page,rows).success(
            function(response){
                $scope.list=response.rows;
                $scope.paginationConf.totalItems=response.total;//更新总记录数
            }
        )
    }
    //条件查询带分页
    //定义一个查询条件对象
    $scope.searchEntity={};
    //条件查询
    $scope.search=function (page,rows) {
        seckillOrderService.search(page,rows,$scope.searchEntity).success(
            function (response) {
                $scope.list=response.rows;
                $scope.paginationConf.totalItems=response.total;//更新总记录数
            }
        )
    }
    // 显示状态
    $scope.status = ["未付款","已付款","未发货","已发货","交易成功","交易关闭","待评价"];
})

File: web-manager/src/main/java/cn/itcast/core/controller/SeckillOrderController.java
package cn.itcast.core.controller;

import cn.itcast.core.pojo.entity.PageResult;
import cn.itcast.core.pojo.seckill.SeckillOrder;
import cn.itcast.core.service.SeckillOrderService;
import com.alibaba.dubbo.config.annotation.Reference;
import org.springframework.web.bind.annotation.RequestBody;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.List;


@RestController
@RequestMapping("/seckillOrder")
public class SeckillOrderController {
   @Reference
    private SeckillOrderService seckillOrderService;

   @RequestMapping("/findAll")
    public List<SeckillOrder> findAll(){
       List<SeckillOrder> orderList = seckillOrderService.findAll();
       return orderList;
   }

    //分页查询
    @RequestMapping("/findPage")
    public PageResult findPage(Integer page, Integer rows){
        PageResult pageResult = seckillOrderService.findPage(page, rows);
        return pageResult;
    }
    //多条件分页查询
    @RequestMapping("/search")
    public PageResult search(Integer page,Integer rows,@RequestBody SeckillOrder seckillOrder){
        PageResult pageResult = seckillOrderService.search(page, rows, seckillOrder);
        return pageResult;
    }
}
